### DISCUSSION TITLE
PING ISSUE

### DISCUSSION THREAD

**Discussion by MohamedAboHelal** [asim97/SecureCodingDojo_DEBI#27] [Q&A]
**Hello guys, I'm having a problem now, I'm trying to ping the website but there is no any response although the website is working well when you try to access it through the browser. Does anyone has an idea about this issue ?**

![ping_issue](https://user-images.githubusercontent.com/42704527/124380857-6ff31d00-dcbf-11eb-9b06-e9d5803a7f69.PNG)

@asim97 
@MohamedZidan097 
@omarhany0

**Comment by msale113** [comment]
I did disable the ICMP replay from the firewall

**Comment by MohamedAboHelal** [comment]
Yeah .. I don't know for sure why you disabled it, may to prevent DOS attack or something like this, But let me  share this great blog about disabling ICMP with you.

https://blog.paessler.com/disabling-icmp-and-snmp-wont-increase-security-but-will-impact-network-monitoring

**Reply by msale113** [reply]
thanks for the link .. this will help us for sure.
if we had any issues regarding our network performance, this may be the problem then.
